homework.md

```diff
@@ -110,4 +108,4 @@ To record what changes you made to the code
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+Slack chanels!
```

Comment:
This is definitely right (Slack channels are useful) but I think the Git cheatsheet is a correct answer too.

Reply:
Thanks for your comment, you are right. I will add that to the answer.